.releaserc.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+// Extra types that generate a patch release
+const typesForPatch = ['docs', 'style', 'refactor', 'perf'];
+
+const releaseConfig = {
+    branches: [
+        'master'
+    ],
+    plugins: [
+        [
+            '@semantic-release/commit-analyzer',
+            {
+                releaseRules: [
+                    ...typesForPatch.map((type) => ({
+                        type,
+                        release: 'patch'
+                    }))
+                ],
+                parserOpts: {
+                    noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES', 'BREAKING']
+                }
+            }
```

Comment:
Nit: A few brief comments explaining what these option overrides are accomplishing and *why* would be good.

Reply:
I'd advocate to keep references to the source of truth, perhaps include links via comments?

Note, the links are included in `docs/releasing.md` doc.

Reply:
Yep, links to the relevant documentation would be good. Even `See docs/releasing.md for details` if you think that's the most relevant.